### ISSUE TITLE
Bug: Changing course structure unmoors comments 

### ISSUE THREAD

**Issue by smangham** [OxfordRSE/gutenberg#270]
As the material has been restructured since last year's SABS training, the student progress and comments have become unstuck and no longer reference their place in the material: https://train.rse.ox.ac.uk/event/5

The site could probably do with some kind of 'migrate material' feature that search-replaces comment parents, e.g. a form with "Migrate from: `software_architecture_and_design` to: `HPC_u.software_architecture_and_design`".

**Comment by alasdairwilson** [comment]
Yeah so this was a conscious decision to make the swapover to the new url structure easier.

I did consider writing an alembic migration script to migrate the old values for this db field to the new system but it wasn't considered critical. If it had just been a column rename or something then fine but to parse the urls to make sure they are the old system and then insert something which we couldnt know in general in advance (because it is based on settings in the config yaml file) seemed like way too much work for a small gain.

A manual database migration would be easier to do and not require being as robust so I think you should just do that.
